### PR TITLE
New Home Page

### PR DESCRIPTION
**Pull request by calebjacob**
Closes: https://github.com/near/near-discovery-components/issues/360

We can safely merge and publish this component without actually affecting `near.org`. The gateway code still uses our hard coded Next JS pages from when we thought we had an SEO issue. Once all the new page BOS components are finished, I'll submit a gateway PR to render the new BOS components.

One change I made in this PR is removing all the CSS variables from `DIG.Theme` to avoid confusion and duplication. Our discovery gateway has the same values copied into `theme.css`, which already provides these variables to the entire Next JS app and BOS components on our gateway. This `theme.css` file is necessary at the Next JS level so our primary navigation components (which are native React/Next components) have access to our theme colors. We can keep thinking through the best way to handle theme colors going forward, but at least for now, this change will help reduce confusion. Let me know if you have any concerns!

### CODE REVIEW

File: src/NearOrg/LatestNews.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+  The following data fetching was pulled from this source code:
```
Comment (charleslavon) [inline src/NearOrg/LatestNews.jsx:2]: 🔥

File: src/NearOrg/TotalStats.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const stats = {
+  totalApps: 0,
+  totalComponents: 0,
+  totalDevelopers: 0,
+};
+const componentsData = Social.keys("*/widget/*", "final", {
```
Comment (charleslavon) [inline src/NearOrg/TotalStats.jsx:6]: nice approach to making this dynamic

**Comment by charleslavon** [comment]
Do we have a reasonable option to improve the centering of these numbers over their labels?
Screen Shot 2023-10-06 at 4 17 27 PM

**Comment by charleslavon** [comment]
awesome work @calebjacob this looks super good

**Review by charleslavon — APPROVED** [review]

**Comment by calebjacob** [comment]
> Do we have a reasonable option to improve the centering of these numbers over their labels? Screen Shot 2023-10-06 at 4 17 27 PM

Good point! I pushed a commit to center those labels on the numbers. Thanks for the feedback! :)

**Pull request merged**